lib/index.js

```diff
@@ -51,6 +51,12 @@ function getSDKOptions () {
       type: Boolean,
       default: false,
       description: 'Specifies to add or not to add README along with sdk'
+    },
+    variables: {
```

Comment:
Where are we passing variables exactly? Here or through the generate function?

Reply:
these are the variables which the user will provide